scripts/prep_forecast.sh

```diff
@@ -66,12 +66,24 @@ ATMPETS=${ATMPETS:-"72"}   #Number of ATM Pets
 # model_configure variables 
 DT_ATMOS=${DT_ATMOS:-900}  #DATM time step [seconds]
 coupling_interval_fast_sec=${coupling_interval_fast_sec:-$DT_ATMOS} #likely can be depricated from DATM
-DATM_FILENAME_BASE=${DATM_FILENAME_BASE:-'cfsr.'} #The prefix of the forcing files for the DATM
-#for cfsr:  (will be different for gefs)
-NFHOUT=${NFHOUT:-6}  #nfhout number of hours between DATM inputs 6 for cfsr 3 for gefs    
-IATM=${IATM:-1760}  #dimension of DATM input files, lon     
-JATM=${JATM:-880}   #dimension of DATM input files, lat
-
+DATM_FILENAME_BASE=${DATM_FILENAME_BASE:-"${FORCING_SRC,,}."} #The prefix of the forcing files for the DATM
+
+#nfhout number of hours between DATM inputs 6 for cfsr 3 for gefs
+#dimension of DATM input files, lon
+#dimension of DATM input files, lat
+
+if [ "${FORCING_SRC,,}" = "cfsr" ]; then
+   NFHOUT=${NFHOUT:-6}
+   IATM=${IATM:-1760}
```

Comment:
This is correct IATM, JATM for 20111001:

ncdump -h /scratch2/NCEPDEV/marineda/godas_input/DATM_INPUT/CFSR/201110/cfsr.2011100206.nc
netcdf cfsr.2011100206 {
dimensions:
	lon = 1760 ;
	lat = 880 ;


However, for another date: 
ncdump -h /scratch2/NCEPDEV/marineda/godas_input/DATM_INPUT/CFSR/199304/cfsr.1993041012.nc
netcdf cfsr.1993041012 {
dimensions:
	lon = 1152 ;
	lat = 576 ;
	time = UNLIMITED ; // (1 currently)

So apparently the dimensions of these files change in time.  Not sure what date that change happens at.

Reply:
Looking at sizes of files, you would need 
   IATM=${IATM:-1760}
    JATM=${JATM:-880}
for the 2011 cases, 
and 
   IATM=${IATM:-1152}
    JATM=${JATM:-576}
for all other dates (currently converted and on hera) for CFSR.